test/fshtypes/FshQuantity.test.ts

```diff
@@ -58,20 +56,18 @@ describe('FshQuantity', () => {
       expect(result).toEqual('100 \'mm\' "Display"');
     });
 
-    // NOTE: Technically this is invalid, but SUSHI never creates a FshQuantity with a non-UCUM system
-    // This is purely for testing purposes
     it('should return string for unit code with non-UCUM system and display', () => {
       const code = new FshCode('bar', 'http://foo.com', 'Display');
       const quantity = new FshQuantity(100, code);
       const result = quantity.toString();
-      expect(result).toEqual('100 \'bar\' "Display"');
+      expect(result).toEqual('100 http://foo.com#bar "Display"');
     });
 
     it('should return string for unit code with code with spaces', () => {
       const code = new FshCode('milli meters');
       const quantity = new FshQuantity(100, code);
       const result = quantity.toString();
-      expect(result).toEqual("100 'milli meters'");
+      expect(result).toEqual('100 #milli meters');
```

Comment:
I think that since this code is 'milli meters' with a space, we actually need to have quotes around the text after the `#`, so I think the expected output should be `100 #"milli meters"`. This made me realize that it might be better to use the `FshCode`'s ``toString` method if the system isn't UCUM. I think the `FshCode.toString` method should account for the quotes around space and should handle the case where there is no system.